### DISCUSSION TITLE
Save contents of display to button

### DISCUSSION THREAD

**Discussion by lageist** [UNC-Project-Open-AAC/OS-DPI#136] [Ideas]
Any way in the current build to simulate saving the contents of the display to a button location? A couple common use cases for this feature: (1) compose a personal story to share in the moment during conversation and wish to save to a button location, to save on composition effort/time when you wish to share again with someone new; (2) participating in a work/educational session and thinking about and concurrently composing questions and comments for the presenter/group and wish to save to button location(s) to support in-time expression when invited.

**Comment by gbishop** [comment]
Yes, this is doable. It might even be doable now. I'll have to check. I'm
pretty sure you can set a variable to the value of your display state. Then
you could speak it anytime you want.

When the display becomes selectable you could use another small display to
contain the saved text. You'd press that small button to speak it (or move
it back to the display).

Lots of variations are possible.

**Reply by lageist** [reply]
Excellent!
